test/DebugInfo/NonSemantic/Shader200/DebugInfoStringType.ll

```diff
@@ -1,5 +1,5 @@
 ; RUN: llvm-as %s -o %t.bc
-; RUN: llvm-spirv -spirv-text %t.bc -o %t.spt --spirv-debug-info-version=nonsemantic-shader-200
+; RUN: llvm-spirv -spirv-text %t.bc -o %t.spt --spirv-debug-info-version=nonsemantic-shader-200 --spirv-ext=+SPV_KHR_non_semantic_info
```

Comment:
No need to enable SPV_KHR_non_semantic_info explicitly for Debug tests - the extension is being enabled implicitly

Reply:
ah nice that landed, fixed, thanks